pkgs/development/libraries/yajl/default.nix

```diff
@@ -2,20 +2,15 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "yajl";
-  version = "unstable-2022-04-20";
+  version = "unstable-2024-02-01";
```

Comment:
```suggestion
  version = "0-unstable-2024-02-01";
```

Reply:
Just looked it up in the contribution manual and it looks like this was changed:

> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.

I'll amend the version.